### PR TITLE
Support cuDNN 5.1

### PR DESCRIPTION
**Pull request by unnonouno**

### CODE REVIEW

File: run_test.py [hunk 1]
```diff
@@ -62,7 +62,7 @@
         conf = {
             'base': 'ubuntu14_py2',
             'cuda': 'cuda65',
-            'cudnn': 'cudnn2',
+            'cudnn': 'cudnn5',
```
Comment (delta2323) [inline run_test.py]: This change does not seems to be related to CuDNN 5.1. Should we include this change in this PR?

Comment (unnonouno) [inline run_test.py]: I want to test all versions of cudnn. cudnn2 is tested in py2 option. Instead cudnn5 in this test.

Comment (delta2323) [inline run_test.py]: You choose CuDNN v5.1 for Py3 and CuDNN v5 for example. Is this because you want to test the former combination?

Comment (unnonouno) [inline run_test.py]: Yes, i want to test v5.1 more precisely than v5.

Comment (unnonouno) [inline run_test.py]: BTW, we may need to discuss which combinations of libraries we need to test.

Comment (delta2323) [inline run_test.py]: I agree with you. For now, it is OK to change the combination to be tested as you do.

**Comment by delta2323** [comment]
LGTM 👍

**Pull request merged**